include/macroni/Common/MLIRUtils.hpp

```diff
@@ -0,0 +1,8 @@
+#pragma once
+
+#include "vast/Util/Common.hpp"
```

Comment:
You want to make your own `Common.hpp` and "import" all the things you want - this way you are vulnerable to our changes - we change something and you need to update all places.

Reply:
I see what you mean.
I can re-alias all the types defined in Macroni's dependencies, but I think I'll create a separate issue and PR just for that change since it will touch many different parts of the project.

Reply:
Yes that's exactly the way to do it :).